docs/developers/Centaur.md

```diff
@@ -159,8 +154,6 @@ or `papi_v2beta_centaur_application.conf`
     2. sum-back: runs summarizer
     3. front-back: exposes HTTP
 - Horicromtal Engine Upgrade: Combination of Horicromtal and Engine Upgrade
-- PAPI Upgrade: Tests run with an older version of Papi and upon restart use a newer version of Papi
-- PAPI Upgrade New Workflows: Test definition [does not run any tests](https://travis-ci.org/broadinstitute/cromwell/jobs/475378412)
```

Comment:
Should these be replaced with Batch rather than removed?

Reply:
If I'm understanding things correctly, the PAPI upgrade tests were checking upgrading from PAPI to PAPI V2 so there isn't a Batch equivalent. (This is separate from the Engine Upgrade that gets the previous cromwell version and does that upgrade)

Reply:
Ah, good call!